invokeai/frontend/web/src/features/lora/store/loraSlice.ts

```diff
@@ -8,7 +8,7 @@ export type Lora = {
 };
 
 export const defaultLoRAConfig: Omit<Lora, 'id' | 'name'> = {
-  weight: 1,
+  weight: 0.75,
```

Comment:
I suggest defaulting LoRA weight to 0.75, as 1.0 is almost always too strong and we use 0.75 as the default in the 2.3 branch